tests/test_saneyaml.py

```diff
@@ -137,6 +137,19 @@ def test_load_ignore_aliases(self):
         ])
         assert  expected == result
 
+    def test_is_iso_date(self):
+        assert saneyaml.is_iso_date('1994-01-01')
+        assert saneyaml.is_iso_date('2004-01-01')
+        assert not saneyaml.is_iso_date('1800-01-01')
+        assert not saneyaml.is_iso_date('2100-01-01')
+        assert saneyaml.is_iso_date('2004-01-30')
+        assert not saneyaml.is_iso_date('2004-01-40')
+        assert not saneyaml.is_iso_date('2004-01-4')
+        assert not saneyaml.is_iso_date('2004-01-3a')
+        assert not saneyaml.is_iso_date('1994-01-1')
+        assert not saneyaml.is_iso_date('1994-1-1')
+        assert not saneyaml.is_iso_date('193')
+
```

Comment:
Hmm, I wonder if I should add this to the end of the tests?
```suggestion
        # Hey, nothing's perfect
        assert saneyaml.is_iso_date('2003-01-29')
        assert saneyaml.is_iso_date('2004-01-35')
```

Reply:
Yes, this is weird, but it is best to capture also this weirdness in tests.